kythe/release/release.BUILD

```diff
@@ -21,12 +21,21 @@ construct_vnames_config(
     ],
 )
 
-# Clone of default Java proto toolchain with "annotate_code" enabled for
-# cross-language metadata file generation.
 proto_lang_toolchain(
     name = "java_proto_toolchain",
-    command_line = "--java_out=annotate_code:$(OUT)",
-    runtime = "@com_google_protobuf//:protobuf_java",
+    command_line = "--java_out=annotate_code,shared,immutable,no_enforce_api_compatibility:$(OUT)",
+    runtime = ":protobuf",
+)
+
+java_library(
+    name = "protobuf",
+    visibility = ["//visibility:private"],
+    exports = [
+        "@com_google_protobuf//:protobuf_java",
+    ],
+    deps = [
+        "@com_google_protobuf//:protobuf_java",
```

Comment:
Does this not need jsr250 annotations as well?

Reply:
Turns out none of the changes in this file are needed. I originally did have them, but didn't finish removing the unnecessaries.

Reply:
So after several experiments, I conclude that this does not need the annotations, however "deps" needed to be "runtime_deps". I'm not quite certain why the presence of this helps. Perhaps it strips out the "Generated" annotation.